src/analytics/analytics.js

```diff
@@ -46,5 +45,9 @@ function identifyUser() {
   }
 }
 
+function sendPageEvent() {
```

Comment:
Nice.  And smart.  I'd like to add a little documentation, but maybe when this moves to frontend-analytics.  Thanks.

FYI: I also confirmed with Data Science that for pages that require authentication, we do **not** need a page event here for the unauthenticated user, and should just log events for what the user sees, which would be an unauthenticated view of the login page with the profile page as the referrer.